src/LsqFit.jl

```diff
@@ -1,9 +1,9 @@
 module LsqFit
 
     export curve_fit,
-           estimate_errors,
```

Comment:
I think we should continue to export `estimate_errors` for one release so that users will see the deprecation warning.

Reply:
Thanks for pointing out! `@deprecate` should already have handled export, according to its [documentation](https://github.com/JuliaLang/julia/blob/60031ad3de561221ed99fe817cb0bb38bb14903c/base/deprecated.jl#L10):

> ```@deprecate ... Remove the definition of the deprecated method and unexport it, as @deprecate takes care of calling the replacement and of exporting the function.```

Reply:
Ah, good to know.